libs/clients/syslumenn/src/lib/__mock-data__/requestHandlers.ts

```diff
@@ -65,9 +71,46 @@ export const requestHandlers = [
       return res(ctx.status(401), ctx.json(VHFAIL))
     }
   }),
+  rest.get(
+    url('/api/SkraningaradiliDanarbus/:nationalid/:id'),
+    (req, res, ctx) => {
+      const success =
+        req.params.nationalid && req.params.nationalid === '0101302399'
+
+      if (success) {
+        return res(ctx.status(200), ctx.json(ESTATE_REGISTRANT_RESPONSE))
+      } else {
+        // TODO: get confirmation from Syslumenn, currently returns 500
+        return res(ctx.status(404))
+      }
+    },
+  ),
   rest.post(url('/api/v1/SyslMottakaGogn'), (req, res, ctx) => {
     return res(ctx.status(200), ctx.json(DATA_UPLOAD))
   }),
+  rest.post(url('/api/VedbokavottordRegluverki'), (req, res, ctx) => {
+    const body = req.body as VedbandayfirlitSkeyti
+    const assetId = body.fastanumer ?? ''
+    const response = VEDBANDAYFIRLRIT_REGLUVERKI_RESPONSE
+    switch ((body.tegundAndlags as number) ?? -1) {
+      case AssetType.RealEstate: {
+        if (!/f?\d+/.test(assetId)) return res(ctx.status(404), ctx.json([]))
+        response[0].heiti = REAL_ESTATE_ADDRESS_NAME
+        return res(ctx.status(200), ctx.json(response))
+      }
+      case AssetType.Vehicle: {
+        // This test will break if tested with Icelandic private license plates, even though they are legal
```

Comment:
Should we do something about this, or will this be addressed at a later point?

Reply:
Is old code from 2 months ago, no longer in use. I have removed it.